content/en/blog/_posts/XX-userns-enabled-by-default/index.md

```diff
@@ -0,0 +1,280 @@
+
+ * _Enablement of new use cases_: User namespaces allow containers to gain
+certain capabilities inside their own user namespace without affecting the host.
+This unlocks new possibilities, such as running applications that require
+privileged operations without granting full root access on the host. This is
+particularly useful for running nested containers.
+
+{{< figure src="/images/blog/2024-04-22-userns-beta/image.svg" alt="Image showing IDs 0-65535 are reserved to the host, pods use higher IDs" title="User namespace IDs allocation" class="diagram-medium" >}}
+
+Without using a user namespace, a container running as root in the case of a
+container breakout has root privileges on the node. If some capabilities
+were granted to the container, the capabilities are valid on the host too. None
+of this is true when using user namespaces (modulo bugs, of course 🙂).
+
+## Demos
+
+Rodrigo created demos to understand how some CVEs are mitigated when user
+namespaces are used. We showed them here before, but take a look if you haven't:
+
+Mitigation of CVE 2024-21626 with user namespaces:
+
+{{< youtube id="07y5bl5UDdA" title="Mitigation of CVE-2024-21626 on Kubernetes by enabling User Namespace support" class="youtube-quote-sm" >}}
+
+Mitigation of CVE 2022-0492 with user namespaces:
+
+{{< youtube id="M4a2b4KkXN8" title="Mitigation of CVE-2022-0492 on Kubernetes by enabling User Namespace support" class="youtube-quote-sm" >}}
+
+## Everything you wanted to know about user namespaces in kubernetes
+
+Here we try to answer some of the questions we have been asked about user
+namespaces support in Kubernetes.
+
+**1. What are the requirements to use it?**
+
+The requirements are documented [here][userns-req]. But we will elaborate a bit
+more, in the following questions.
+
+[userns-req]: /docs/concepts/workloads/pods/user-namespaces/#before-you-begin
+
+**2. What is idmap mounts and why the file-systems used need to support it?**
+
+Idmap mounts is a Linux kernel feature that uses a mapping of UID/GIDs when
+accessing a mount. When combined with user namespaces, it greatly simplifies the
+support for volumes, as you can forget about the host UID/GIDs the user
+namespace is using.
+
+In particular, thanks to idmap mounts we can:
+ * Run each pod with different UIDs/GIDs on the host. This is key for the
+   lateral movement prevention we mentioned earlier.
+ * Share volumes with pods that don't use user namespaces.
+ * Enable/disable user namespaces without needing to chown the pod's volumes.
+
+However, support for idmap mounts in the kernel is not black/white. The support
+is per file-system and support was added over time to different file-systems.
+
+You can check which kernel version added support for each file-system in the
+`mount_setattr` man page, or checking the online version of it [here][mount_setattr].
+
+Most popular file-systems are supported in some kernel version, the notable
+absence that isn't supported yet is NFS.
```

Comment:
Did you really mean "some" kernel version? Or something like below could be better?

```suggestion
You can find most popular file-systems are supported in recent kernel versions. One of the most notable
absence here is NFS, which isn't supported yet as of writing.
```

Reply:
recent is not accurate, if you check the link you will see that a lot of file-systems are supported since 5.12 that was released ~4 years ago

Reply:
Thanks for the clarification! In that case, I still think it's misleading to say "in some kernel version", because it's not just about one version. Recent may be a bit misleading indeed, so it may be better to be more specific, with something like "You can find most popular file-systems have been supported in kernel versions from as early as 5.12"?

Reply:
I don't see it unclear, as just a few lines above we explained that support for different fs was added in different kernel releases. But that is my way of reading it, I might be biased because I wrote the text.

Given that we already explained that support is per file-system in this same question, a few lines above, I think it is simpler to just say:

```
Most popular file-systems are supported, the notable absence that isn't supported yet is NFS
```

Changed to that. But feel free to re-open if it isn't clear for you